lib/ViewModels/DataCatalogTabViewModel.js

```diff
@@ -32,6 +33,18 @@ var DataCatalogTabViewModel = function(options) {
     this.svgArrowDown = defaultValue(options.svgArrowDown, svgArrowDown);
     this.svgArrowRight = defaultValue(options.svgArrowRight, svgArrowRight);
     this.svgInfo = defaultValue(options.svgInfo, svgInfo);
+
+    this.panelHasFocus = knockout.observable(false);
+
+    var that = this;
+
+    knockout.getObservable(this, 'isActive').subscribe(function(newValue) {
+        if (newValue) {
+            window.setTimeout(function(){
```

Comment:
Why is this `setTimeout` necessary?  In my (limited) testing it seems to work ok without it.